packages/dotcom-build-sass/package.json

```diff
@@ -25,9 +25,10 @@
     "css-loader": "^3.0.0",
     "cssnano": "^4.1.10",
     "dlv": "^1.1.3",
+    "fibers": "^4.0.2",
```

Comment:
Is this required or does it just help to speed things up?

Reply:
The latter. It is detailed in the second of the links in the PR description.

Reply:
Does it seem to be effective?

Reply:
So far I have only used it to build the `ft-ui` example which does not have much to build so I will find out when I start testing it on Page Kit-consuming apps.

Reply:
Testing with `fibers` for `next-myft-page` build:
```
real	0m20.085s
user	0m22.182s
sys	0m1.758s
```

Testing without `fibers` for `next-myft-page` build:
```
real	0m18.221s
user	0m21.776s
sys	0m1.603s
```

🤷‍♂ I will remove it.

Reply:
Removed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/764/commits/e062b2c147733d6354a2edbee31d750a17bbc404).